CHANGELOG.md

```diff
@@ -13,7 +12,8 @@
 -----------
 
 ### Internals
-* None.
+* Illegal freeing of in-file-memory is now detected when freeing is
```

Comment:
We are missing the assertion fix. So I think we should add this to the public "Enhancements" section:
* Added asserts and better detection of illegal frees to get more details about possible file corruptions. This can be observed with assertions including "Invalid mnemonic." or "Top ref outside file".

Reply:
What is the assertion fix? The assertions you mention are not affected by the changes we have made.